report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,38 +1,60 @@
+The model outputs are the quantified chemical compositions of geological samples. 
+These are primarily the concentrations of major elements, represented in weight percentage. 
+While background elements are also present, our current analysis does not quantify these. 
+Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $n_o$ denotes the number of target oxides in the Concentration Matrix $\mathbf{C}[\chi, o]$.
+This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^{n_o}$ of estimated oxide concentrations. The vector $\mathbf{x}$ is derived from matrix $\mathbf{M}$, representing processed spectral data:
 \[
 \mathbf{v} = \mathcal{F}(\mathbf{x}).
 \]
 
+As previously introduced, quantifying chemical compositions from \gls{libs} spectral data poses significant challenges, including high dimensionality and multicollinearity of the data, as well as matrix effects. Here, we further examine these challenges:
+
+\begin{itemize}
+    \item \textbf{Data Dimensionality and Collinearity:} The high dimensionality of spectral data, coupled with multicollinearity—where multiple spectral features exhibit strong correlations—complicates the modeling and analysis. 
+    This challenge arises both in the Intensity Matrix $\mathbf{I[\chi, l, s, \lambda]}$ and consequently the Averaged Intensity Matrix $\mathbf{A[\chi, l, \lambda]}$, where the high number of wavelengths leads to a large number of dimensions, and the intensity readings across these wavelengths often display high correlations.
+    For instance, emission lines from the same element can overlap, causing their intensity readings at different wavelengths to be interdependent\cite{andersonImprovedAccuracyQuantitative2017}. 
+    This interdependency complicates the extraction of independent spectral features necessary for accurate quantitative analysis.
+
+    \item \textbf{Matrix Effects:} As noted in the introduction~\ref{sec:introduction}, matrix effects refer to any effect that can cause variations in the intensity of emission lines of an element independent of the concentration of that element. 
+    These effects are caused by the complex interplay of various physical processes within the plasma generated by the \gls{libs} technique. The presence of different background elements can significantly alter the emission intensities by affecting these processes.
+    This also affects both the Intensity Matrix $\mathbf{I[\chi, l, s, \lambda]}$ and the Averaged Intensity Matrix $\mathbf{A[\chi, l, \lambda]}$, where the recorded intensities are influenced by the interaction of multiple physical processes, such as the coupling process between the laser photons and the target, self-absorption of optical emission lines within the plasma or collisional interactions in the plasma.
```

Comment:
```suggestion
   \item \textbf{Matrix Effects:} As noted in the introduction~\ref{sec:introduction}, matrix effects refer to any effect that can cause variations in the intensity of emission lines of an element independent of its concentration. These effects arise from the complex interplay of various physical processes within the plasma generated by the \gls{libs} technique. The presence of different background elements can significantly alter emission intensities by affecting these processes. This impacts the accuracy of our previously discussed mapping function $\mathcal{F}$, which quantifies chemical compositions from processed \gls{libs} signals. Variations in recorded intensities due to matrix effects complicate the extraction of accurate and independent spectral features because the overlapping and interaction of emission lines can mask or distort the true signal of individual elements. This makes it difficult to isolate the contributions of each element to the overall spectrum. Consequently, it becomes challenging to precisely map the processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^{n_o}$ of estimated oxide concentrations, thereby impacting the reliability of the predicted chemical compositions~\cite{cleggRecalibrationMarsScience2017, andersonImprovedAccuracyQuantitative2017}.
```